src/applications/vaos/appointment-list/redux/reducer.js

```diff
@@ -57,6 +57,7 @@ const initialState = {
   facilitySettingsStatus: FETCH_STATUS.notStarted,
   facilitySettings: null,
   backendServiceFailures: null,
+  badAppointmentId: false,
```

Comment:
nit: looks like this is no longer necessary.

Reply:
Thanks. Removed.